### DISCUSSION TITLE
what is tubeup doing on the second download?

### DISCUSSION THREAD

**Discussion by 111100001** [bibanon/tubeup#288] [General]
![image](https://github.com/bibanon/tubeup/assets/24229327/ad673b10-7d69-429c-a2a9-6ad6dc664bf2)

**Comment by 111100001** [comment]
the second download is faster than the first one, but i dont understand what it is actually doing

**Comment by vxbinaca** [comment]
Video and audio are split on some extractors, like youtube. We instruct yt-dlp to get the best quality video and audio even if thats not whats normally streamed, and then to mux them together.
